Resources/doc/configuration.rst

```diff
@@ -722,6 +722,32 @@ the ORM resolves to:
 There are lots of other configuration options that you can use to overwrite
 certain classes, but those are for very advanced use-cases only.
 
+Oracle
+=======
+
+If the environment format configured in oracle does not meet doctrine requirements,
+you need to use OracleSessionInit listener so that doctrine is aware of the format used by oracle.
+
+You can do so easily with
+
+.. code-block:: yaml
+
+    services:
+        oracle.listener:
+            class: Doctrine\DBAL\Event\Listeners\OracleSessionInit
+            tags:
+                - { name: doctrine.event_listener, event: postConnect }
+
+The environment variables needed for doctrine and their default format are:
```

Comment:
These are environment vars for Oracle DB, not Doctrine DBAL: the [listener](https://github.com/doctrine/dbal/blob/b551fcb26a81e230085c0dca88174573b6c36609/lib/Doctrine/DBAL/Event/Listeners/OracleSessionInit.php) doesn't really use the environment, AFAIK.